### DISCUSSION TITLE
hexo-theme-butterfly 魔改与美化

### DISCUSSION THREAD

**Discussion by giscus** [youngjunning/blog.zisheng.pro#112] [Announcements]
# hexo-theme-butterfly 魔改与美化

本文将介绍本站对 hexo-theme-butterfly 进行的一些美化和魔改。

https://youngjuning.js.org/3f81e8c6ddd1/

**Comment by AkimioJR** [comment]
请问页脚上面的那一栏是这么魔改的，就是和安知鱼主题挺像的那个样式
